src/Umbraco.Web/HealthCheck/Checks/Config/LoadBalancingHealthCheck.cs

```diff
@@ -0,0 +1,509 @@
+            var actions = new List<HealthCheckAction>();
+            //finally for Slave and Master configurations we can look up in the umbracoServer table to see what other servers are in play...
+
+            var serverRegistrationService = ApplicationContext.Current.Services.ServerRegistrationService;
+            // wonder if it's useful to see what the ServerRegistrationService returns for role
+            var currentServerRole = serverRegistrationService.GetCurrentServerRole();
+            var currentServerIdentity = serverRegistrationService.CurrentServerIdentity;
+            resultMessage = "Identity: " + currentServerIdentity + " | Role: " + currentServerRole;
+
+            return
+               new HealthCheckStatus(resultMessage)
+               {
+                   ResultType = resultType,
+                   Actions = actions
+               };
+        }
+        /// <summary>
+        /// List out active servers from umbracoServer database table
+        /// </summary>
+        /// <returns></returns>
+        private HealthCheckStatus DisplayFlexibleLoadBalancingServers()
+        {
+            string resultMessage = "";
+            StatusResultType resultType = StatusResultType.Info;
+            // can we fix anything
+            var actions = new List<HealthCheckAction>();
+            //finally for Slave and Master configurations we can look up in the umbracoServer table to see what other servers are in play...
+
+            var serverRegistrationService = ApplicationContext.Current.Services.ServerRegistrationService;
+            var currentServerRole = serverRegistrationService.GetCurrentServerRole();
+            var activeServers = serverRegistrationService.GetActiveServers();
+
+            if (activeServers.Any())
+            {
+                //how many servers
+                if (activeServers.Count() == 1)
+                {
+                    resultMessage = "There is only one active Server in the Flexible Load Balancing Pool";
+                }
+                else
+                {
+                    resultMessage = "There are " + activeServers.Count() + " active servers in the Flexible Load Balancing Pool";
+                }
+                //is it appropriate to use html here?
+                resultMessage = resultMessage + "<ul>";
+                foreach (var server in activeServers)
+                {
+
+                    resultMessage = resultMessage + "<li>";
+                    resultMessage = resultMessage + server.ServerAddress + " | " + server.ServerIdentity + " | Registered: " + server.Registered + " | Master: " + server.IsMaster + " | Last Accessed: " + server.Accessed;
+                    resultMessage = resultMessage + "</li>";
+                }
+                resultMessage = resultMessage + "</ul>";
+
+            }
+            else
+            {
+                resultMessage = "0 Active servers in the Flexible Load Balancing Pool";
```

Comment:
Maybe this message could say that it is possible that explicit master slave election is in place : https://our.umbraco.com/documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/flexible-advanced#explicit-master-scheduling-server